spectrochempy/analysis/mcrals.py

```diff
@@ -96,17 +98,35 @@ class MCRALS(DecompositionAnalysis):
         5, help="Maximum number of successive non-converging iterations."
     ).tag(config=True)
 
+    solverConc = tr.Enum(
+        ["lstsq", "nnls", "pnnls"],
+        default_value="lstsq",
+        help=(
+            """
+              Solver used to solve :math:`C @ S^t = X` for :math:`C`\ .
```

Comment:
If you use triple quote the following lines MUST start at the first column!!! If not, docstrings are not valid and scrubbed up in the sphinx documentation. 
This is why I changed most of them with single quote which as also the advantage to let sphinx managing the line breaks.

Reply:
Actually I could may be write a routine to make the docstring dedent automatically. I will try.

Reply:
Additional there is some syntax problem.

look at this:  

  solverConc = tr.Enum(
        ["lstsq", "nnls", "pnnls"],
        default_value="lstsq",
        help=(
            """
              Solver used to solve :math:`C @ S^t = X` for :math:`C`\ .

              - ``'lstsq'`` (default) : uses ordinary least squares with `~numpy.linalg.lstsq`
              - ``'nnls'``\ : Non-negative least squares (`~scipy.optimize.nnls`\ ) are applied sequentially on all
              profiles
              - ``'pnnls'``\ : non-negative least squares (`~scipy.optimize.nnls`\ ) are applied on profiles indicated in
              ``'nonnegConc'`` and ordinary least squares on other profiles.
            """
        ),
    ).tag(config=True)

it gives:
<img width="571" alt="image" src="https://user-images.githubusercontent.com/2428474/235282839-98abad9f-ca25-4023-93de-e0fbb9bf7370.png">

So, this is wrong:

              - ``'nnls'``\ : Non-negative least squares (`~scipy.optimize.nnls`\ ) are applied sequentially on all
              profiles

it should be (see the alignment of the second line)

              - ``'nnls'``\ : Non-negative least squares (`~scipy.optimize.nnls`\ ) are applied sequentially on all
                 profiles

Additional personally, I try to always keep the length of line at below the Black/flake8 default which is 88. 
So for docstrings except the first short summary line (in normal docstrings) which should fit in a single line, I propose to make line breaks, if you agree or I can change the black and flake8 default?.

I would write (note the r""" which is useful to remove pytest deprecation warnings regarding the backslash - note that sphinx don't care about):

    solverConc = tr.Enum(
        ["lstsq", "nnls", "pnnls"],
        default_value="lstsq",
        help=(
            r"""
            Solver used to solve :math:`C @ S^t = X` for :math:`C`\ .

            - ``'lstsq'`` (default) : uses ordinary least squares with
              `~numpy.linalg.lstsq`
            - ``'nnls'``\ : Non-negative least squares (`~scipy.optimize.nnls`\ ) are
              applied sequentially on all profiles
            - ``'pnnls'``\ : non-negative least squares (`~scipy.optimize.nnls`\ ) are
              applied on profiles indicated in ``'nonnegConc'`` and ordinary least
              squares on other profiles.
            """
        ),
    ).tag(config=True)

which gives
<img width="598" alt="image" src="https://user-images.githubusercontent.com/2428474/235283138-7fb827be-28d1-4cc3-95ee-c197c80f96a1.png">

Finally note that if the string start right after the """ a dedent must be applied to the following lines.

    solverConc = tr.Enum(
        ["lstsq", "nnls", "pnnls"],
        default_value="lstsq",
        help=(
            r"""Solver used to solve :math:`C @ S^t = X` for :math:`C`\ .

    - ``'lstsq'`` (default) : uses ordinary least squares with
       `~numpy.linalg.lstsq`
    - ``'nnls'``\ : Non-negative least squares (`~scipy.optimize.nnls`\ ) are
       applied sequentially on all profiles
    - ``'pnnls'``\ : non-negative least squares (`~scipy.optimize.nnls`\ ) are
       applied on profiles indicated in ``'nonnegConc'`` and ordinary least
       squares on other profiles.  """
        ),
    ).tag(config=True)

Reply:
I think thé doctrine are OK nox.